src/main/groovy/net/neoforged/gradleutils/GradleUtilsExtension.groovy

```diff
@@ -65,13 +75,18 @@ abstract class GradleUtilsExtension {
         }
         this.gitInfo = objects.mapProperty(String, String)
                 .convention(rawInfo.map { it.gitInfo })
+
+        shouldSign.convention(project.provider { (System.getenv('GPG_PRIVATE_KEY') || System.getenv('GPG_SUBKEY')) || new File(project.gradle.gradleUserHomeDir, 'secring.gpg').exists() })
```

Comment:
Why are we checking for `secring.gpg` under the Gradle user home directory?

Reply:
That should be checking for the `signing.secretKeyRingFile` property, whoops.